Actions/MediaFastForwardAction.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Threading.Tasks;
+using Windows.Media.Control;
+using SuchByte.MacroDeck.ActionButton;
+using SuchByte.MacroDeck.Plugins;
+
+
+// ReSharper disable once CheckNamespace
+namespace MediaControls_Plugin; // Don't change because of compatibility
+
+public class MediaFastForwardAction : PluginAction
+{
+    private GlobalSystemMediaTransportControlsSessionManager _manager;
+
+    public MediaFastForwardAction()
+    {
+        if (PluginManager.Plugins.ContainsKey("Macro Deck Media Controls"))
+        {
+            MacroDeckPlugin plugin = PluginManager.Plugins["Macro Deck Media Controls"];
+            if (plugin.GetType().IsEquivalentTo(typeof(MediaControlsPlugin)))
+            {
+                _manager = ((MediaControlsPlugin)plugin).Manager;
+            }
+        }
```

Comment:
You don't need to make this that complicated, you can make a public static instance of the plugin in the MediaControlsPlugin class and then access it.

Example: https://github.com/manuelmayer-dev/Macro-Deck-Windows-Utils-Plugin/blob/master/Main.cs